HOWTO

```diff
@@ -2096,6 +2096,10 @@ with the caveat that when used on the command line, they must come after the
 	When hipri is set this determines the probability of a pvsync2 I/O being high
 	priority. The default is 100%.
 
+.. option:: nowait : [pvsync2] [libaio] [io_uring]
+
+	Set RWF_NOWAIT on I/O, do not wait if request cannot be executed immediately.
```

Comment:
Agree on those suggestions. Would also probably prefer a single commit, since that single commit could then have an explanation of what it is. The commit messages seem a bit messed up right now, and are either empty (io_uring), or some implementation detail (aio), or no explanation of the feature (sync).

So either make the first one have a full explanation, with the other referencing it, or combine them into one commit instead and include the whole thing. Either approach is fine with me.